src/logic/function/basic.lean

```diff
@@ -404,6 +404,11 @@ def involutive {α} (f : α → α) : Prop := ∀ x, f (f x) = x
 lemma involutive_iff_iter_2_eq_id {α} {f : α → α} : involutive f ↔ (f^[2] = id) :=
 funext_iff.symm
 
+/-- Involuting an `ite` of an involuted value `x : α` negates the `Prop` condition in the `ite`. -/
+lemma inv_ite {α} (P : Prop) [decidable P] {f : α → α} (h : involutive f) (x : α) :
```

Comment:
Given the lemma above uses `involutive` and not `inv`, it would make sense to me to call this
```suggestion
lemma involutive_ite {α} (P : Prop) [decidable P] {f : α → α} (h : involutive f) (x : α) :
```
This also avoids confusing with `inverse`

Reply:
Actually, this might even be moved inside the `namespace` below (and renamed to `protected lemma ite_not`?) so that dot notation can be used.

Reply:
@pechersky what do you think of renaming as in my comment so that dot notation can be used?

Reply:
Yeah, about to push that commit